third_party/3/pyspark/sql/_typing.pyi

```diff
@@ -32,7 +32,7 @@ AtomicDataTypeOrString = Union[pyspark.sql.types.AtomicType, str]
 DataTypeOrString = Union[pyspark.sql.types.DataType, str]
 OptionalPrimitiveType = Optional[PrimitiveType]
 
-RowLike = TypeVar("RowLike", List[Any], Tuple[Any, ...], pyspark.sql.types.Row)
+RowLike = TypeVar("RowLike", List[Any], Tuple[Any, ...], pyspark.sql.types.Row, None)
```

Comment:
After some thought I don't think this the way to go. Instead, annotation `emptyRDD` should be changed to

```python
    def emptyRDD(self) -> RDD[Any]: ...
```

Reply:
You're right, that seems like a better way to do it. It's more consistent, as calling `collect` on any RDD results in a list of a priori undefined type (unlike Scala's typed Datasets), so Any is a better fit. Fixing…